src/design-guidelines/disabled-hidden/examples/disabled-vs-hidden.tsx

```
@@ -20,17 +20,25 @@ export class ButtonDisabledVsHideExample {
     @State()
     public required: boolean;
 
+    @State()
+    private doNotSubmitClicked: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)